locales/de.json

```diff
@@ -2648,14 +2648,15 @@
 			"atSchool": "an der Schule",
 			"atYourSchool": "an deiner Schule ",
 			"created": "erstellt",
-			"from": "von",
+			"by": "von ",
+			"closingBracket": ")",
```

Comment:
not a translation

Reply:
in english we have "created by John Doe" and in German "erstellt von John Doe"